consumer/Dockerfile

```diff
@@ -1,16 +1,13 @@
 FROM python:2.7-slim
 
 WORKDIR /app
-ADD /common/. /app
-ADD /subscriber/init_consumer.py /app
+ADD . /app
+ADD ./thrift/kafka_message.thrift /app
```

Comment:
Not sure if this step is required since you regenerating thrift interface at build time below (by calling `RUN thrift -r --gen py kafka_message.thrift`)